app/models/report_form.rb

```diff
@@ -7,17 +8,47 @@ class ReportForm
     attr_accessor "development_#{n}"
   end
 
+  (1..SMART_OBJECTIVES).each do |n|
+    attr_accessor "smart_what_#{n}"
+    attr_accessor "smart_how_#{n}"
+  end
+
   def development_as_json
     (1..DEVELOPMENT_OBJECTIVES).map { |n| send("development_#{n}") || '' }
   end
 
+  def smart_as_json
+    (1..SMART_OBJECTIVES).map do |n|
+      {
+        what: send("smart_what_#{n}") || '',
+        how: send("smart_how_#{n}") || ''
+      }
+    end
+  end
+
   def self.from_report(report)
     report_form = ReportForm.new
 
-    report.development.each_with_index do |d, index|
-      report_form.send("development_#{index + 1}=", d)
-    end
+    process_development(report, report_form)
+    process_smart(report, report_form)
 
     report_form
   end
+
+  def self.process_smart(report, report_form)
+    if report.smart
```

Comment:
Is there any way that we can make Report return sensible defaults, so that we don't need all these null checks? (I'm thinking of `send("smart_what_#{n}") || ''` as well, here).